new/LIT6373CanonAgreement.xml

```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6373CanonAgreement" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">በእንተ፡ ፹ወ፩መጻሕፍት፡ ከመ፡ ፩ቃል፡ እማንቱ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">bʾǝnta samānǝyā wa ʾaḥadu maṣaḥeft kama ʾaḥadu qal ʾǝmantu</title>
+                <title xml:lang="en" corresp="#t1">Agreement of the Eighty-One Canonical Scriptures</title>
```

Comment:
Perhaps you could get the translation a little closer to the original text.

Reply:
I've had a go, let me know what you think